bin/internal/update_dart_sdk.ps1

```diff
@@ -44,7 +44,16 @@ Import-Module BitsTransfer
 Start-BitsTransfer -Source $dartSdkUrl -Destination $dartSdkZip
 
 Write-Host "Unzipping Dart SDK..."
-Expand-Archive $dartSdkZip -DestinationPath $cachePath
+if (Get-Command Expand-Archive -errorAction SilentlyContinue) {
+    # Expand-Archive requires PowerShell 5 or newer
+    Expand-Archive $dartSdkZip -DestinationPath $cachePath
```

Comment:
Expand-Archive is the recommended (and only official) way to expand archives in a PowerShell script. The alternative I provide here for older PowerShells is kinda hacky, as it calls out to the Windows GUI to do the unzipping (it will actually pop up a new GUI window displaying the unzipping status).